.github/workflows/ks-py-coding-style.yml

```diff
@@ -6,54 +6,29 @@ on:
   pull_request:
     paths:
       - '.github/workflows/ks-py-coding-style.yml'
+      - 'backend/kesaseteli/**'
+      - 'backend/shared/**'
   workflow_dispatch:
 
 env:
   DEBUG: 1
 
 jobs:
-  py-coding-style:
-    name: Python coding style
+  pre-commit:
+    name: Run pre-commit hooks
     runs-on: ubuntu-latest
-
     steps:
-      - name: Check out repository
-        uses: actions/checkout@v3
-
-      - name: Set up Python 3.9
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.9'
-
-      - name: Cache pip packages
-        uses: actions/cache@v3
-        env:
-          cache-name: cache-pip-modules
-        with:
-          path: ~/.pip-cache
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/requirements.txt') }}-${{ hashFiles('**/requirements-dev.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
-
-      - name: Install dependencies
-        run: cd backend/kesaseteli && pip install -r requirements.txt -r requirements-dev.txt
-
-      - name: (KS) Formatting
-        run: cd backend/kesaseteli && black --check .
-
-      - name: (KS) Linting
-        run: cd backend/kesaseteli && flake8
-
-      - name: (KS) Import sorting
-        run: cd backend/kesaseteli && isort -c .
-
-      - name: (Shared) Formatting
-        run: cd backend/shared && black --check .
-
-      - name: (Shared) Linting
-        run: cd backend/shared && flake8
-
-      - name: (Shared) Import sorting
-        run: cd backend/shared && isort -c .
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Setup Python ${{ inputs.python-version }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ inputs.python-version }}
+    - name: Run pre-commit (benefit)
```

Comment:
```suggestion
    - name: Run pre-commit (kesaseteli)
```

Reply:
fixed in 8713e06ac1fe3995d126b035d4c58b749e13ee93